app/src/main/java/org/simple/clinic/AppDatabase.kt

```diff
@@ -18,8 +18,8 @@ import org.simple.clinic.home.overdue.OverdueAppointment_Old
 import org.simple.clinic.medicalhistory.Answer
 import org.simple.clinic.medicalhistory.MedicalHistory
 import org.simple.clinic.monthlyReports.questionnaire.Questionnaire
-import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireLayout
 import org.simple.clinic.monthlyReports.questionnaire.QuestionnaireType
+import org.simple.clinic.monthlyReports.questionnaire.component.BaseComponentData
```

Comment:
@siddh1004 This DSL is for a generic questionnaire, not specifically for `monthlyReports`.
Also, there's no data here, so maybe call it `BaseComponent` instead of `BaseComponentData`?

Finally, it would look like: 
```org.simple.clinic.questionnaire.component.BaseComponent```
Thoughts?

Reply:
> @olttwa monthlyReports package is to distinguish from other part of the app which uses the same server side rendering technique.

@siddh1004 MonthlyReports should be a package only while rendering. Since these classes are for parsing JSON, they should be generic as Cancer Screening, Monthly drug stock can use them too.

Reply:
> BaseComponent would be another class created for expoxy models. SoData is attached to this.

This makes sense. Maybe rename to `BaseComponentState` (No strong opinions here, choose what you like)

Reply:
@olttwa The naming convention was picked from epoxy documentation.